Dockerfile

```diff
@@ -1,18 +1,17 @@
 FROM registry.access.redhat.com/ubi8/ruby-27
 
 USER root
-RUN dnf --setopt=skip_missing_names_on_install=False,tsflags=nodocs --save \
-  && rpm -Uvh https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm \
+RUN rpm -Uvh https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm \
   && dnf update -y \
-  && dnf remove -y postgresql \
-  && dnf install -y shared-mime-info postgresql12 postgresql12-devel postgresql12-libs \
+  && dnf remove -y postgresql* \
+  && dnf install --setopt=skip_missing_names_on_install=False,tsflags=nodocs -y shared-mime-info postgresql12-12.13 postgresql12-devel-12.13 postgresql12-libs-12.13 \
```

Comment:
Version 12.14 depends on `perl(IPC::Run)` which is not in the standard repos.
Need to install epel for this.

We need to tackle this at some point anyway, because we need to upgrade to Postgres 13 (see [THREESCALE-8545](https://issues.redhat.com/browse/THREESCALE-8545)), and postgres13 depends on it too.

Reply:
we would maybe need to use CentOS Stream 8 in such case (in the future_)